### DISCUSSION TITLE
RoBERTa coming to Spark NLP?

### DISCUSSION THREAD

**Discussion by ericbugin** [JohnSnowLabs/spark-nlp#2865] [Ideas]
Hey, just wondering if RoBERTa is coming to Spark NLP? Tried looking for it but didn't see it. Might be a nice addition!

**Comment by maziyarpanahi** [comment]
Hi,
Yes, we might bring RoBERTa into Spark NLP in the future releases.

**Reply by ericbugin** [reply]
Hey, I believe you guys added RoBERTa, thanks for that! Just to make sure, will this not be available to use until Spark-NLP 3.1 releases? I noticed the RoBertaEmbeddings annotator is not working yet.

**Reply by maziyarpanahi** [reply]
Those features haven't been released. We update the docs/README on each release, anything is not there is under the development. I don't have any ETA on 3.1 release currently.
